xmake/modules/package/tools/meson.lua

```diff
@@ -307,7 +307,7 @@ function _get_configs(package, configs, opt)
     end
 
     -- add cross file
-    if package:is_cross() then
+    if package:is_cross() or package:is_plat("mingw") then
```

Comment:
is_cross should be true for mingw, we need not add it.

https://github.com/xmake-io/xmake/blob/6b400aa1f7a578694079ccb84813a02192cb28e6/xmake/core/package/package.lua#L693

plat: mingw
host: windows
subhost: msys

Reply:
It false for mingw because it goes here
https://github.com/xmake-io/xmake/blob/6b400aa1f7a578694079ccb84813a02192cb28e6/xmake/core/package/package.lua#L689-L691

Reply:
ok